wiki/redirect.yaml

```diff
@@ -213,7 +213,7 @@
 "xk":                          "Game_modifier/xk"
 
 "acc":                         "Accuracy"
-"accuracy_glossary":           "Accuracy"
+"accuracy":                    "Accuracy"
```

Comment:
this is unnecessary, osu! wiki picks up any case

Reply:
I had forgotten to remove it, as for `nc`